### PR TITLE
[TM-1047] add alert for check polygons

### PR DESCRIPTION
**Pull request by egrojMonroy**
https://gfw.atlassian.net/browse/TM-1047

### CODE REVIEW

File: src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx [hunk 1]
```diff
@@ -3,29 +3,66 @@ import { useEffect, useState } from "react";
...
   };
-
+  const { mutate: getValidations } = usePostV2TerrafundValidationSitePolygons({
+    onSuccess: () => {
+      setShouldRefetchValidation(true);
+      setClickedValidation(false);
+      displayNotification(
+        t("Please update and re-run if validations fail."),
+        "success",
+        "Success! TerraMatch reviewed the polygon"
```
Comment (roguenet) [inline src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx:49]: This title needs translation

File: src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx [hunk 2]
```diff
@@ -3,29 +3,66 @@ import { useEffect, useState } from "react";
...
+      );
+    },
+    onError: () => {
+      setClickedValidation(false);
+      displayNotification(t("Please try again later."), "error", "Error! TerraMatch could not review polygons");
```
Comment (roguenet) [inline src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx:54]: Title needs translation here too

File: src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx [hunk 3]
```diff
@@ -40,6 +41,17 @@ const CheckPolygonControl = (props: CheckSitePolygonProps) => {
...
+  const [notificationStatus, setNotificationStatus] = useState<{
+    open: boolean;
+    message: string;
+    type: "success" | "error" | "warning";
+    title: string;
+  }>({
+    open: false,
+    message: "",
+    type: "success",
+    title: "Success!"
```
Comment (roguenet) [inline src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx:53]: Translation for titles here and on lines 91 and 96

File: src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx [hunk 4]
```diff
@@ -88,17 +126,21 @@ const CheckPolygonControl = (props: CheckSitePolygonProps) => {
...
   return (
     <div className="grid gap-2">
+      <Notification {...notificationStatus} />
       <div className="rounded-lg bg-[#ffffff26] p-3 text-center text-white backdrop-blur-md">
         <Button
           variant="text"
           className="text-10-bold my-2 flex w-full justify-center rounded-lg border border-tertiary-600 bg-tertiary-600 p-2 hover:border-white"
-          onClick={() => setClickedValidation(true)}
+          onClick={() => {
+            console.log("Clcededede ", true);
```
Comment (roguenet) [inline src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx:141]: 🛅

File: src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx (hunk 1 above)
Comment (egrojMonroy) [reply src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx:49]: done

File: src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx (hunk 2 above)
Comment (egrojMonroy) [reply src/components/elements/Map-mapbox/MapControls/CheckIndividualPolygonControl.tsx:54]: done

File: src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx (hunk 3 above)
Comment (egrojMonroy) [reply src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx:53]: done

File: src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx (hunk 4 above)
Comment (egrojMonroy) [reply src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx:141]: removed

**Review by cesarLima1 — APPROVED** [review]

**Review by roguenet — APPROVED** [review]

**Pull request merged**